service/dap/server_test.go

```diff
@@ -3778,7 +3778,7 @@ func TestAttachRequest(t *testing.T) {
 	})
 }
 
-func TestPauseAndContinue(t *testing.T) {
+func TestUnsupportedCommandResponses(t *testing.T) {
```

Comment:
Why was this test function renamed?

Reply:
This was a rebase merge error, will fix along with the rest the feedback (probably on the weekend as I'm swamped this week by work and college)